### ISSUE TITLE
Trigger associated submit button when "enter" pressed

### ISSUE THREAD

**Issue by mitchhentges** [rust-native-ui/libui-rs#71]
When configuring inputs on a webpage, I can associate them with a form (which has a primary `submit` button). Then, when I click "enter" while any of those inputs are focused, then the submit button will automatically be triggered.

In my case, I've a couple inputs and a submit button:
![Screenshot from 2019-05-25 18-36-38](https://user-images.githubusercontent.com/7784737/58372359-35352d00-7f1c-11e9-8207-011de3a05299.png)

I'd love to make it so hitting "enter" while the `Entry` is focused will cause the submit action :smile:

**Comment by NoraCodes** [comment]
I don't think this is possible using `libui` itself at the moment, sorry! `libui` does have forms, which we will get soon, but no way to set a submit button. It's definitely something I'd like to have, though, so I'll leave the issue open.
